src/mumble/ConnectDialog.cpp

```diff
@@ -1245,11 +1245,17 @@ void ConnectDialog::on_qaFavoritePaste_triggered() {
 }
 
 void ConnectDialog::on_qaUrl_triggered() {
-	ServerItem *si = static_cast< ServerItem * >(qtwServers->currentItem());
-	if (!si || si->qsUrl.isEmpty())
+	auto *si = static_cast< const ServerItem * >(qtwServers->currentItem());
+	if (!si || si->qsUrl.isEmpty()) {
 		return;
+	}
 
-	QDesktopServices::openUrl(QUrl(si->qsUrl));
+	const QStringList allowedSchemes = { QLatin1String("http"), QLatin1String("https") };
+
+	const auto url = QUrl(si->qsUrl);
+	if (allowedSchemes.contains(url.scheme())) {
+		QDesktopServices::openUrl(url);
+	}
```

Comment:
```suggestion
	if (allowedSchemes.contains(url.scheme())) {
		QDesktopServices::openUrl(url);
	} else {
		// Inform user that the requested URL has been blocked
		QMessageBox msgBox;
		msgBox.setText(QObject::tr("Blocked URL scheme \"%1\"").arg(url.scheme()));
		msgBox.setInformativeText(QObject::tr("The URL uses a scheme that has been blocked for security reasons. The requested URL is \"%1\"").arg(url.toString()));
		msgBox.exec();
	}
```

I think we should inform the user about what is going on as otherwise it might cause the impression of Mumble simply not working correctly.